src/utils/configuration.ts

```diff
@@ -0,0 +1,33 @@
+import * as dotenv from 'dotenv';
+
+export class Configurations {
```

Comment:
👍 
we may have to remove serverless-dotenv if we use dotenv.
I think serverless is planning to add support out of the box for it and will be deprecating the serverless-dotenv plugin soon

Reply:
I will have look at it.

Reply:
In order to remove we need serverless to be >= 3.0.0. More details on this page https://www.serverless.com/plugins/serverless-dotenv-plugin